### PR TITLE
Move livereload, image, and favicon to config

### PR DESCRIPTION
**Pull request by codydaig**
Redo of Pull #664 (was having git issues)

### CODE REVIEW

File: config/env/development.js [hunk 1]
```diff
@@ -60,5 +60,6 @@ module.exports = {
 				pass: process.env.MAILER_PASSWORD || 'MAILER_PASSWORD'
 			}
 		}
-	}
+	},
+  livereload: true
```
Comment (rhutchison) [inline config/env/development.js]: fix indentation

Comment (codydaig) [inline config/env/development.js]: That page is massively messed up in terms of spacing. Even certain lines are mixed between spacing and tabs so Sublime is confused. Should be fixed now.

**Comment by lirantal** [comment]
Ok with me
@rhutchison we good here?

**Comment by rhutchison** [comment]
@lirantal his commit is good, I just want to point out indentation issue

https://github.com/codydaig/mean/blob/pull664/config/env/development.js#L54-L63

Can be fixed as a separate PR

**Comment by lirantal** [comment]
yeah that's a separate issue indeed so I'm merging in.

**Pull request merged**